app/templates/power-ups/intro.html

```diff
@@ -10,7 +10,7 @@
   <h3>Power-Up Samples</h3>
   <p>Trello has built two sample Power-Ups that you can use to get started developing right away. These samples demonstrate the capabilities of the platform and should help you to get started.</p>
   <div>
-    <a href="https://glitch.com/#!/project/rune-friend" target="_blank" rel="noopener noreferrer"><button class="mod-primary">Remix the Glitch Sample Project</button></a>
+    <a href="https://glitch.com/#!/project/trello-power-up" target="_blank" rel="noopener noreferrer"><button class="mod-primary">Remix the Glitch Sample Project</button></a>
```

Comment:
@bentleycook  Seems they changed the URL structure. This now needs to be: `https://glitch.com/edit/#!/trello-power-up`